types/base_contract.ts

```diff
@@ -35,7 +42,7 @@ export class BaseContract {
         };
         if (_.isUndefined(txDataWithDefaults.gas) && !_.isUndefined(estimateGasAsync)) {
             const estimatedGas = await estimateGasAsync(txData);
-            txDataWithDefaults.gas = estimatedGas;
+            txDataWithDefaults.gas = new BigNumber(estimatedGas);
```

Comment:
What was the need for the change here?

Reply:
this is just what was being done in the setprotocol.js side so decided to do it here